### DISCUSSION TITLE
Translate Documents Into Chinese

### DISCUSSION THREAD

**Discussion by tubring** [filamentphp/filament#2474] [General]
Hello @danharrin ,

I ran into this project occasionally and just had a try, and found it is amazing. You are doing a great job!

I found there is no Chinese Version in your Document. I was thinking of transalte these documents into Chinese and build a site for that. Just to check if there is any limit(eg, lience issue) for such thing. Thank you,

Regards,
Martin

**Comment by danharrin** [comment]
Hi,

No issue! I think it would be too much work to host Chinese docs on our own website, but feel free to host them externally!
